conf/db/migration/default/V2__create_candidates_table.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE credentials
+(
+    id    uuid primary key,
```

Comment:
What's the reason for using `uuid` instead of `serial`? I'm just curious 😄 .

Reply:
I need to have the ID at app level so that I can use it as a foreign key in the `candidates` table. Using serial I would need to do an additional select.